app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RelationshipsController < ApplicationController
+  before_action :require_login
+
+  def index
+    @following_relationships = current_user.following_relationships
+  end
+
+  def create
+    user = User.find(params[:leader_id])
+    if current_user.can_follow?(user)
+      Relationship.create(follower: current_user, leader: user)
```

Comment:
Thanks Brandon, very good point, which hadn't occurred to me. Change implemented.